foundations/html_css/intro-to-html-css.md

```diff
@@ -18,6 +18,7 @@ If you want to get technical, HTML and CSS aren't _programming languages_ becaus
 This section contains helpful links to other content.  It isn't required, so consider it supplemental or if you need to dive deeper into something.
 
 * [This FreeCodeCamp article](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) goes into a little more depth than the assigned one. It covers things we'll be teaching explicitly in later lessons though, so don't worry about memorizing any of the details.
+* Bookmark [DevDocs.io](https://devdocs.io). Read the “Welcome” message. Massive API documentation collection that even works offline. Essential collection of reference material for everything covered and more. (Maintained by [FreeCodeCamp](https://freecodecamp.org))
```

Comment:
I feel like this is shoehorned here given we don’t mention MDN which should be the standard, and isn’t HTML/CSS specific.